docs/blog/2022-05-02-arch-linux-installation-guide.mdx

```diff
@@ -0,0 +1,355 @@
+Here's where we'll be managing system configurations, in our case we're doing this as an American user,
+for those of you reading outside the US, feel free to set language and keyboard bindings here to a format that works for your machine.
+
+CHROOT into the new system:
+
+    arch-chroot /mnt/
+
+Set the hostname:
+
+    echo <YOUR-HOSTNAME> > /etc/hostname
+
+Set locale:
+
+    echo LANG=en_US.UTF-8 > /etc/locale.conf
+
+Uncomment the following rows of /etc/locale.gen:
+
+    en_US.UTF-8.UTF-8
+
+Generate locale:
+
+    locale-gen
+
+Define hosts in /etc/hosts:
+
+    <ip-address>	<hostname.domain.org>	<hostname>
+    127.0.0.1	<YOUR-HOSTNAME>.localdomain	<YOUR-HOSTNAME>
+    ::1		localhost.localdomain	localhost
+
+Set the root user's password:
+(Set a solid password as you will be signing into the machine as root for post install configuration)
+
+    passwd
+
+## Initramfs
+Configure the creation of initramfs by editing /etc/mkinitcpio.conf. Edit the line HOOKS=... to:
+
+    HOOKS=(base keyboard udev autodetect modconf block keymap encrypt btrfs filesystems resume)
+
+:::note
+resume in the above hooks is for those of you setting up a swap partition. If you don't intend to use swap, you can simply remove it.
+:::
+
+Recreate initramfs:
+
+    mkinitcpio -p linux
+
+## Boot Manager
+
+Install systemd-boot:
+
+    bootctl --path=/boot install
+
+Create file /boot/loader/entries/arch.conf and fill it with:
+
+    title Arch Linux
+    linux /vmlinuz-linux
+    initrd /intel-ucode.img
+    initrd /initramfs-linux.img
+    options cryptdevice=UUID=<UUID-OF-ROOT-PARTITION>:luks:allow-discards root=/dev/mapper/luks rootflags=subvol=@ rd.luks.options=discard rw
+
+:::note
+Optional - If you are using a swap add the following to the options flag. You wall also need to follow this guide to calculate your [offset](https://wiki.archlinux.org/title/Power_management/Suspend_and_hibernate#Hibernation_into_swap_file_on_Btrfs)
+:::
+    resume=/dev/mapper/luks resume_offset=<YOUR-OFFSET>
+
+Now we need to provide the UUID of our root partition
+
+    blkid -s UUID -o value /dev/sda2
+
+Replace the placeholder for UUID-OF-ROOT in /boot/loader/entries/arch.conf
+
+Edit file /boot/loader/loader.conf and fill it with:
+
+    default  arch.conf
+    timeout  4
+    console-mode max
+    editor   no
+
+Lastly, we'll exit chroot, unmount our partition and reboot.
+
+    exit
+    umount -R /mnt
+    reboot
+
+## Congratulations, you now have Arch running on your laptop!
+You now have an encrypted partition of Arch Linux running on your laptop with a root user and networking.
+
+## What's left to do?
+- Sign in as root on your laptop
+- Setup a user with sudo privileges
+- Setup a Window Manager / Desktop Environment
+- Install YAY and AUR
+- Generate SSH Keys
+- Generate GPG Keys
+- Find ways to interject into conversations that you now use Arch
+
+### References
+- [Nerd Stuff Guide](https://nerdstuff.org/posts/2020/2020-004_arch_linux_luks_btrfs_systemd-boot/)
```

Comment:
I do see you referenced it but.. we owe him so much we should probably put it up top

Reply:
agreed